src/contentmanager.cpp

```diff
@@ -189,6 +189,24 @@ QString ContentManager::downloadBook(const QString &id)
     return QString::fromStdString(download->getDid());
 }
 
+void ContentManager::pauseBook(const QString& id)
+{
+    qInfo() << "pause";
+    auto& b = mp_library->getBookById(id);
+    kiwix::Download* download;
+    download = mp_downloader->getDownload(b.getDownloadId());
```

Comment:
We need to call `d->updateStatus(true);`
The `getDownload` method do not call it for us.

And as we are asynchrone here, the download may be complete before we get there (and because we are refreshing the download status only every second). But I'm not sure it is a problem, probably aria will do nothing and keep the download as completed and next refresh will show the correct status.